src/backend/common/DefaultMemoryManager.cpp

```diff
@@ -37,20 +42,21 @@ void DefaultMemoryManager::cleanDeviceMemoryManager(int device) {
     // This vector is used to store the pointers which will be deleted by
     // the memory manager. We are using this to avoid calling free while
     // the lock is being held because the CPU backend calls sync.
-    vector<af_buffer_info> free_ptrs;
+    vector<void*> free_ptrs;
     size_t bytes_freed           = 0;
-    memory::memory_info &current = memory[device];
+    DefaultMemoryManager::memory_info &current = memory[device];
```

Comment:
nit: A lot of these refactors in this file could have been avoided via auto + using statement at the top.